cli/main.go

```diff
@@ -43,11 +43,10 @@ func buildVersionString() string {
 			fmt.Fprintf(versionsWriter, "\t%s\n", image)
 		}
 	}
-	fmt.Fprint(versionsWriter, "\n")
-	fmt.Fprintf(versionsWriter, "reference values for %s platform:\n", platforms.AKSCloudHypervisorSNP.String())
-	fmt.Fprintf(versionsWriter, "\truntime handler:\tcontrast-cc-%s\n", manifest.TrustedMeasurement[:32])
```

Comment:
Runtime handler isn't printed anymore.

Reply:
Because there's no singular valid runtime handler anymore

Reply:
That was already anticipated, that's why there was a platform-specific section in the output.

Reply:
So do you want to roll back to how it was before, but showing everything for SNP and TDX?

Reply:
I'm just saying you shouldn't throw away the work other developers put in there without a thought. Should users construct the runtime name themselves in the future and count the offset of the prefix we are using? Is the length of the offset documented anywhere?

Reply:
I'm fine with changing it back, but then I'd also suggest we don't print the JSON at all. Are you fine with "reverting" that @burgerdev?

Reply:
Sounds like we need to discuss the question of runtime handlers vs. reference values first - we want this to be a 1:1 mapping, right? So, should it go into `ReferenceValues` fields?

Reply:
Implicitly, I'd say it's already there due to the presence of `TrustedMeasurement` in each of the `ReferenceValues`. But I think this is something so adjacent to the general discussion of how we want to handle these that we should discuss it in the sync today.

Reply:
Created [a ticket](https://dev.azure.com/Edgeless/Edgeless/_backlogs/backlog/Edgeless%20Team/Features?workitem=4464) for this